CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### 0.11.2 (Next)
 
 * Your contribution here.
+* [#110](https://github.com/slack-ruby/slack-ruby-bot-server/pull/110): Update the sample_app_activerecord Gemfile and README to make them function as expected.
```

Comment:
Needs a format fix (include your name).

I would also just write "Fix ActiveRecord sample app", short and concise